dataset/copy-fault-tests/invalid-row/vPersonInvalidValueAtEOF.csv

```diff
@@ -0,0 +1,2 @@
+0,4
+2,2147483650
```

Comment:
newline

Reply:
the test case here is invalid value at EOF so there's intentionally no newline